src/models/Database.ts

```diff
@@ -56,6 +56,13 @@ export default class Database {
     return db;
   }
 
+  public notify(): void {
```

Comment:
Few questions:
1. Will `TransactionHandler` call this?
2. If only `message` is updated, why to notify all?  It should only call `this.messageRepository.notify();`, similarly for other entities.

Reply:
1. With the current architecture, I assume yes.
2. This was an optimization that came later in my commits.